bridgy/inventory/__init__.py

```diff
@@ -22,10 +22,20 @@ def inventory(config):
 
     for source, srcCfg in config.sources():
         if source == 'aws':
-            if os.path.exists(os.path.expanduser("~/.aws")):
-                inventory = AwsInventory(cache_dir=config.inventoryDir(AwsInventory.name))
+            cache_dir = config.inventoryDir(AwsInventory.name)
+            if srcCfg['name'] != None:
+                cache_dir = os.path.join(cache_dir, srcCfg['name'])
+                if not os.path.exists(cache_dir):
```

Comment:
Similarly, path creations have been reserved for the `config.create()` method https://github.com/wagoodman/bridgy/blob/master/bridgy/config/base.py#L56